pkg/controller/user-cluster-controller-manager/resources/controller.go

```diff
@@ -223,6 +223,7 @@ type reconciler struct {
 	versions          kubermatic.Versions
 	caBundle          resources.CABundle
 	userClusterMLA    UserClusterMLA
+	cloudProvider     mctypes.CloudProvider
```

Comment:
I think it would be better to use CloudProvider consts defined in KKP, rather than MC types. The type and consts are defined here: https://github.com/kubermatic/kubermatic/blob/67e94dee12b1b471f6ad2563d64304e6088e9c32/pkg/crd/kubermatic/v1/preset.go#L27-L69

(thanks to @xrstf for finding this :slightly_smiling_face:)

Reply:
Cool, I looked around but I didn't find those consts, gonna use them instead.